### PR TITLE
[fix] raciness with status get

### PR DESCRIPTION
**Pull request by jcrugzz**
## Summary

Certain status events can be very close in proximity time. It's feasible that multiple events are trying to create a Status and StausHead record, overwriting the update that happens in very close time proximity as well. We shall see if this is able to solve the issue.

## Changelog
Use. `ConsistentRead` option for. `StatusHead.findOne` and `Status.findOne`

## Test Plan
Tests should still pass

### CODE REVIEW

**Comment by decompil3d** [comment]
Note that strongly consistent reads cost more capacity units than eventually consistent ones. Probably fine, but make sure you are handling capacity properly (I think you're on pay-per-use anyway, right?)

**Comment by jcrugzz** [comment]
@decompil3d yea we use pay per use so we will just pay the extra cost

File: status-handler.js [hunk 1]
```diff
@@ -64,8 +64,8 @@ class StatusHandler {
     const ev = this._transform(data, 'event');
     const [, head, current] = await Promise.all([
       StatusEvent.create(ev),
-      StatusHead.findOne(ev),
-      Status.findOne(ev)
+      StatusHead.findOne(ev, { ConsistentRead: true }),
```
Comment (msluther) [inline status-handler.js]: Should we delay the call to statusHead until we figure out if there is a `current`? So we're not always reading a record we won't need most of the time (unless we're trying to create a new head).

Comment (jcrugzz) [reply status-handler.js]: @msluther we could but making this function complete faster actually doesnt help our current situation 😭

Comment (jcrugzz) [reply status-handler.js]: but i guess it could save us money in this new aws world

Comment (msluther) [reply status-handler.js]: Yeah, I was thinking about saving the extra call / cost / speed. Wouldn't block on it though.

**Pull request merged**